instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteAlternativesTest.kt

```diff
@@ -144,6 +147,19 @@ class RouteAlternativesTest : BaseCoreNoCleanUpTest() {
                     mapboxNavigation.getAlternativeMetadataFor(it)
                 )
             }
+
+            val mockedAlternativesResponse = InputStreamReader(
+                openRawResource(context, R.raw.route_response_alternative_continue)
+            ).use {
+                DirectionsResponse.fromJson(it)
+            }
+            newAlternatives.forEach {
+                assertEquals(
+                    "some info was lost during NN -> Nav SDK transition",
```

Comment:
?

Reply:
it means that info from response doesn't match info we get from NN

Reply:
What does it have to do with the new annotation?